### PR TITLE
Implementation for Thermostat Website

### PR DESCRIPTION
**Pull request by zkysar**
Hey, guys so this is what a PR looks like. These are really good for managing large code bases. The basic premiss here is that I will link all of you with your Github IDs like so:

Reviewers: @Rattrocker @anthonygraca @AgentChicken @silvan9145  @dave-luk 

When you see this, try not to just merge it. The basic process here is for ya'll to look at the code and if you think its good give a :thumbsup: and that will let me know you've seen it n'such. Feel free to try out some of the cool features of git here. For instance you can comment on a particular line of code. You can use labels. Of course....
# YOU CAN USE MARKDOWN

So go ahead and mess around in here if you want. I'll merge this in a couple days unless you guys have any issues with my PR :worried:
# David was here

### CODE REVIEW

**Comment by zkysar** [comment]
👍

**Comment by silvan9145** [comment]
Am I doing this right?

File: Thermostat/web-service/readme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Running the Thermostat Lesson
+1. Spin up an ec2 instance
```
Comment (Rattrocker) [inline Thermostat/web-service/readme.md]: let em rip

File: Thermostat/download-and-run.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+echo "Updating System Resources"
+sudo yum -y update
+
+echo "Installing Git"
```
Comment (zkysar) [inline Thermostat/download-and-run.sh:4]: Hey your code sucks.

Comment (zachrank) [reply Thermostat/download-and-run.sh:4]: I agree

File: Thermostat/download-and-run.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+sudo yum -y install git
```
Comment (zachrank) [inline Thermostat/download-and-run.sh:5]: I’m Rick Harrison and this is my pawn shop. I work here with my old man and my son, Big Hoss, and in 23 years I’ve learned one thing. You never know what is gonna come through that door.

**Comment by zkysar** [comment]
@silvan9145

**Pull request merged**